modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/PRMForm/components/fields/DragAndDrop/DragAndDrop.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ClayInput} from '@clayui/form';
+import {FormikContextType} from 'formik';
+import Dropzone, {useDropzone} from 'react-dropzone';
+
+import MDFClaim from '../../../../../interfaces/mdfClaim';
+import PRMFormFieldProps from '../common/interfaces/prmFormFieldProps';
+import PRMFormFieldStateProps from '../common/interfaces/prmFormFieldStateProps';
+
+interface IProps {
+	onAccept: (value: File[]) => void;
+}
+
+const DragAndDrop = ({
+	description,
+	field,
+	label,
+	onAccept,
+	required,
+}: PRMFormFieldProps &
+	PRMFormFieldStateProps<File[]> &
+	Pick<FormikContextType<MDFClaim>, 'setFieldValue'> &
+	IProps) => {
+	const {acceptedFiles, getInputProps, getRootProps, open} = useDropzone({
```

Comment:
```suggestion
	IProps) => {
	
	const {acceptedFiles, getInputProps, getRootProps, open} = useDropzone({
```

Atençao com os espaçamentos